.github/workflows/cpflow-promote-staging-to-production.yml

```
@@ -0,0 +1,197 @@
+          missing_vars="$(comm -23 <(echo "${staging_vars}") <(echo "${production_vars}"))"
+
+          if [[ -n "${missing_vars}" ]]; then
+            echo "::error::Production is missing environment variables that exist in staging"
+            echo "${missing_vars}"
+            exit 1
+          fi
+
+      - name: Capture current production image
+        id: capture-current
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:9:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)